manager/controllers/system_advisories.go

```diff
@@ -77,6 +77,11 @@ func SystemAdvisoriesHandler(c *gin.Context) {
 		return
 	}
 
+	if !(database.IsValidUUID(inventoryID)) {
```

Comment:
You can omit these external brackets - use just `if !utils.IsValidUUID(inventoryID) {`.